ocfweb/docs/docs/staff/getinvolved.md

```diff
@@ -0,0 +1,236 @@
+   * Configure Firefox to show the "are you sure you want to quit" dialog
+     ([PR #373][puppet-373])
+   * Allow operations staff to refund pages ([PR #368][puppet-368])
+   * Monitor our mirrors to see if they’re out of date
+     ([healthcheck script][mirror-healthcheck],
+     [configuring server to call the healtcheck
+     script][mirror-healthcheck-puppet],
+     [Prometheus rules for alerting][prometheus-mirror])
+   * Monitor printers to see if they’re out of paper, toner, or jammed
+     ([printer.rules.yaml][prometheus-printer])
+   * Send a desktop notification if a print jobs fails ([PR #321][puppet-321])
+* [ocfweb][ocfweb] ([issues][ocfweb/issues]) - the source code for our website,
+  www.ocf.berkeley.edu. Written in Python using the Django framework.
+   * Hosts our documentation for users and staffers (including
+     [this page][getinvolved-src]!) ([docs][docs-src])
+   * JSON APIs for lab hours ([api/hours.py][api-hours])
+   * [Account creation][account-register], [password resets][account-chpass],
+     [others][account]
+   * Lab usage statistics ([stats/][ocfweb-stats])
+   * Form for requesting a virtual host ([account/vhost.py][ocfweb-vhost])
+* [ocflib][ocflib] ([issues][ocflib/issues]) - a Python library installed on all
+  servers. Has Python functions for various parts of the lab infrastructure.
+   * Account management, searching ([account/][ocflib-account])
+   * Accessing the database ([infra/mysql.py][ocflib-mysql])
+   * Send automated emails ([misc/mail.py][ocflib-mail])
+   * Printer querying and statistics ([printing/][ocflib-printing])
+* [utils][utils] ([issues][utils/issues]) - similar to ocflib, this contains
+  Python scripts that are run by humans.
+   * Paper quota refund ([printing/paper][utils-paper])
+   * Query/search OCF accounts ([acct/][utils-acct])
+   * For users: create MySQL database and HTTP webdir
+     ([makeservices/makemysql-real][utils-makemysql-real],
+     [makeservices/makehttp][utils-makehttp])
+   * Create new virtual machines ([staff/sys/makevm][utils-makevm])
+   * Wake up desktops from sleep ([staff/lab/lab-wakeup][utils-lab-wakeup])
+* [slackbridge][slackbridge] ([issues][slackbridge/issues]) - bridges our IRC
+  network with our Slack organization, so staff members can exercise choice in
+  what chat program they’d like to use. Written in Python.
+* [ircbot][ircbot] ([issues][ircbot/issues]) - a fun chatbot, also used to
+  approve new account requests. Written in Python.
+   * See https://ircbot.ocf.berkeley.edu/ for a listing of IRC bot functionality
+
+#### TODOs
+
+If you’re feeling particularly adventurous, you can [search for the string
+“TODO” across our entire codebase][sourcegraph-todo].
+
+#### Help requests
+
+During the semester, our [[help@ocf.berkeley.edu mailing list|doc contact]]
+gets around 14 emails per week from the UC Berkeley community. These emails get
+responded to by volunteers like you. Most requests are about requesting virtual
```

Comment:
Sentence is awkwardly phrased. How about: "Volunteers like you respond to these emails"